ci/gpu/java.sh

```diff
@@ -95,37 +89,18 @@ conda activate rapids
 # gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
 # gpuci_mamba_retry install -y "your-pkg=1.0.0"
 
-
-gpuci_logger "Check compiler versions"
-python --version
-
-
 gpuci_logger "Check conda environment"
 conda info
 conda config --show-sources
 conda list --show-channel-urls
 
-function install_dask {
-    # Install the main version of dask, distributed, and streamz
-    gpuci_logger "Install the main version of dask, distributed, and streamz"
-    set -x
-    pip install "git+https://github.com/dask/distributed.git@main" --upgrade --no-deps
-    pip install "git+https://github.com/dask/dask.git@main" --upgrade --no-deps
-    # Need to uninstall streamz that is already in the env.
-    pip uninstall -y streamz
-    pip install "git+https://github.com/python-streamz/streamz.git@master" --upgrade --no-deps
-    set +x
-}
-
 ################################################################################
 # INSTALL - Install libcudf artifacts
 ################################################################################
 
 gpuci_logger "Installing libcudf & libcudf_kafka"
 gpuci_mamba_retry install -c ${CONDA_ARTIFACT_PATH} libcudf libcudf_kafka
 
-install_dask
```

Comment:
Comment not about this line but the line 124 above: 
```
gpuci_logger "Installing libcudf & libcudf_kafka"
```
Should ` & libcudf_kafka` also be removed?

Reply:
Good catch. I fixed this in the GitHub Actions PR already. Will fix.